### PR TITLE
More cruft identified by CLion

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

File: src/server/input/key_repeat_dispatcher.cpp [hunk 1]
```diff
@@ -152,19 +152,16 @@ bool mi::KeyRepeatDispatcher::dispatch(std::shared_ptr<MirEvent const> const& ev
...
-        if (!handle_key_input(mir_input_event_get_device_id(iev), mir_input_event_get_keyboard_event(iev)))
-            return next_dispatcher->dispatch(event);
-        else
-            return true;
+        return next_dispatcher->dispatch(event);
     }
     return next_dispatcher->dispatch(event);
 }
 
 // Returns true if the original event has been handled, that is ::dispatch should not pass it on.
```
Comment (wmww) [inline src/server/input/key_repeat_dispatcher.cpp:163]: Comment is no longer applicable

**Review by wmww — CHANGES_REQUESTED** [review]

File: tests/mir_test_doubles/CMakeLists.txt [hunk 2]
```diff
@@ -20,8 +20,7 @@ set(
   triggered_main_loop.cpp
   fake_alarm_factory.cpp
   ${PROJECT_SOURCE_DIR}/tests/include/mir/test/doubles/null_message_sender.h
-  ${PROJECT_SOURCE_DIR}/tests/include/mir/test/doubles/mock_message_sender.h
-  ${PROJECT_SOURCE_DIR}/tests/include/mir/test/doubles/mock_console_services.h
+        ${PROJECT_SOURCE_DIR}/tests/include/mir/test/doubles/mock_console_services.h
```
Comment (wmww) [inline tests/mir_test_doubles/CMakeLists.txt:23]: indentation

File: src/server/input/key_repeat_dispatcher.cpp (hunk 1 above)
Comment (AlanGriffiths) [reply src/server/input/key_repeat_dispatcher.cpp]: It was never applicable, but gone now

**Review by wmww — APPROVED** [review]
bors r+

**Pull request merged**